### DISCUSSION TITLE
Update Emby

### DISCUSSION THREAD

**Discussion by TurboTronix** [tteck/Proxmox#2552] [General]
Will there be an update for Emby server? Latest version is 4.8...

**Comment by tteck** [comment]
To update, follow the instructions,

**Reply by tteck** [reply]
![Screenshot 2024-02-23 3 36 06 PM](https://github.com/tteck/Proxmox/assets/21342670/1496a15b-4ac3-4907-92f9-3cb4f937d433)

**Reply by TurboTronix** [reply]
Hmm i did and got the same results, I'll try again once i get home today.

**Reply by TurboTronix** [reply]
It worked, issue was I did not run it in the actual console...
